MoMMI/Modules/github.py

```diff
@@ -881,7 +837,45 @@ def format_desc(desc: str) -> str:
     embed.set_footer(
         text=f"{repo}#{content['number']} by {content['user']['login']}", icon_url=content["user"]["avatar_url"])
 
-    embed.description = format_desc(content["body"])
+    embed.description = format_desc(content["body"]) + "\n"
+
+    merge_sha = prcontent["merge_commit_sha"]
+    check_content = await get_github_object(f"/repos/{repo}/commits/{merge_sha}/check-runs")
+
+    #get the admemes to add icons for all the checks so we can do this prettier
+    for check in check_content["check_runs"]:
+        status = "❓"
+        if check["status"] == "queued":
+            status = "😴"
+        elif check["status"] == "in_progress":
+            status = "🏃"
+        elif check["status"] == "completed":
+            if check["conclusion"] == "neutral": #would sure be nice to just know these huh GITHUB
+                status = "😐"
+            else:
+                con = check["conclusion"]
+                status = f"add {con}"
+
+        cname = check["name"]
+        embed.description += f"`{cname} {status}`\n" #will only need \n as long as we got no icons
```

Comment:
Discord's embeds actually have a way to add separate fields like this you can use.

https://discordpy.readthedocs.io/en/latest/api.html#embed